src/Tokenizer/Analyzer/ClassAnalyzer.php

```diff
@@ -0,0 +1,132 @@
+<?php
```

Comment:
I will leave it as a blocker here
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4310/files#r271581322